app/src/main/res/values/styles.xml

```diff
@@ -79,4 +79,22 @@
     <item name="android:textSize">14sp</item>
     <item name="android:clickable">false</item>
   </style>
+
+  <style name="AppTabLayout" parent="Widget.Design.TabLayout">
+    <item name="tabIndicatorColor">@color/white</item>
+    <item name="tabIndicatorHeight">4dp</item>
+    <item name="tabPaddingStart">6dp</item>
+    <item name="tabPaddingEnd">6dp</item>
+    <item name="tabBackground">?attr/selectableItemBackground</item>
+    <item name="tabTextAppearance">@style/AppTabTextAppearance</item>
+    <item name="tabIconTint">@color/tab_icon_color_selector</item>
+    <item name="tabSelectedTextColor">@color/white</item>
+  </style>
+  <!-- for text -->
```

Comment:
I think we don't need this comment, the style name is self explanatory.

Reply:
Done